setup.py

```diff
@@ -0,0 +1,25 @@
+from setuptools import setup, find_packages
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setup(
+    name="cosmocrat-cli",
+    author="MC",
+    author_email="mapcolonies@gmail.com",
+    description="osm tools wrapper cli",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/MapColonies/cosmocrat-cli",
+    packages=find_packages(exclude=["tests"]),
+    install_requires=["osmeterium==0.0.3", "mapcoloniesjsonlogger==1.0.0"],
```

Comment:
what if we need more packages than that? we should read `requirements.txt` which can be generated with `Pipfile`

Reply:
we can leave it for now if it is too complicated

Reply:
with [pipenv-setup](https://github.com/Madoshakalaka/pipenv-setup) we can sync the pipenv packages with setup.py install_requires list.
simply using `pipenv-setup sync --pipfile`